app/pages/_app.tsx

```diff
@@ -4,34 +4,34 @@ import Head from "next/head";
 import "../styles/globals.scss";
 
 const metaTags = {
-  title: "Auckland DHB Vaccination Rates to 90%",
+  title: "Aotearoa New Zealand Vaccination Rates to 90% - outoflockdown.co",
   description:
-    "Easily view Auckland's vaccinations to 90% by DHB and the daily progress towards getting out of lockdown.",
+    "View vaccination rates for each District Health Board in New Zealand and track the daily progress towards the 90% goal, enabling Kiwis to get out of lockdown.",
```

Comment:
Sorry minor wording tweak again

```suggestion
    "View vaccination rates for each District Health Board (DHB) in New Zealand and track daily progress towards the 90% goal, enabling Kiwis to get out of lockdown.",
```